jobs/blog_validation_errors.py

```diff
@@ -22,49 +22,40 @@
 from jobs.types import base_validation_errors
 import utils
 
-from typing import Any # pylint: disable=unused-import
```

Comment:
This file was filled with usages of unexplained Any. So I decided to **move** this file to the denylist, removed type annotations with unexplained Any, but kept the remaining annotations (removed type ignores also, since they are unnecessary - they can be readded in future if we really need them). We can add better type annotations to this and other `jobs/` files modified in this PR once the base_jobs and parent class are type annotated.